fjohnson-instacartlite/app/controllers/orders_controller.rb

```diff
@@ -0,0 +1,34 @@
+class OrdersController < ApplicationController
+
+    before_action :set_user
+
+    def index
+        @orders = @user.orders.all
+    end
+
+    def show
+        @order = @user.orders.find(params[:id])
```

Comment:
Your error message might be coming from the fact that the order id you're passing in isn't an order that belongs to the `current_user`

Try `@order = Order.find(params[:id])` here.

Reply:
I created an orders method in the users controller and added the proper route and view. I also updated the show method in the orders controller to @order = Order.find(params[:id]).

I'm now getting this error: "Couldn't find Order with 'id'=:order_id"